plugins/SitesManager/API.php

```diff
@@ -1066,28 +1065,22 @@ public function setGlobalExcludedUserAgents($excludedUserAgents)
      * only the global user agent substrings (see @setGlobalExcludedUserAgents) will be used.
      *
      * @return bool
+     * @deprecated
```

Comment:
could we change the comment to something like `@deprecated Will be removed in Matomo 4.0`?